### PR TITLE
docs: Added local setup and dev steps

### PR DESCRIPTION
**Pull request by saideepesh000**
Issue #42 

Suggest any changes and add-ons if I can add...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,6 +205,18 @@ Run blazepack in verbose mode and share the log by raising an issue
 ```
 blazepack start --verbose > blazepack.log
 ```
+### Contributing To This Project
+
+```
+Fork this repository to make your own copy
```
Comment (ameerthehacker) [inline README.md:211]: add only the commands inside the backticks and keep other content out of it for better readability

File: README.md [hunk 2]
```diff
@@ -205,6 +205,18 @@ Run blazepack in verbose mode and share the log by raising an issue
...
+git clone https://github.com/yourusername/blazepack
+cd blazepack
+npm install
+npm link
```
Comment (ameerthehacker) [inline README.md:215]: actually you can create a playground folder (which is git ignored) inside the codebase and use `node ../bin/blazepack.js create my-cra` inside the playground folder also to test things as npm link would mess with your local installation of blazepack and it is very flaky.

**Review by ameerthehacker — CHANGES_REQUESTED** [review]
Great work, have added few minor changes, do let me know if you need any clarifications around them.

File: README.md (hunk 2 above)
Comment (saideepesh000) [reply README.md:215]: Nice idea!!

File: README.md [hunk 3]
```diff
@@ -207,16 +207,16 @@ blazepack start --verbose > blazepack.log
...
 cd blazepack
 npm install
-npm link 
-# With this, blazepack project is added to global packages, 
-# i.e we can make changes to our local blazepack project and see the changes without having to rebuild the package
-# Make any changes to the project and instantly see the changes via blazepack <command> <package>
+mkdir playground # Creates a git-ignored playground folder 
+cd playground
+node ../bin/blazepack.js <command> <name> <template> # Executes our local blazepack project in playground folder
 ```
```
Comment (saideepesh000) [inline README.md:218]: Hey @ameerthehacker, I not sure how to frame the sentences exactly in this context, please make necessary changes.

**Review by ameerthehacker — APPROVED** [review]

**Comment by ameerthehacker** [comment]
@all-contributors please add @saideepesh000 for docs

**Comment by ameerthehacker** [comment]
@saideepesh000 thanks for the contribution

**Pull request merged**